core/tests/test_utils.py

```diff
@@ -396,9 +519,11 @@ def set_collection_editors(self, collection_editor_usernames):
             self.set_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
     def get_current_logged_in_user_id(self):
+        """Returns the login user id"""
```

Comment:
Missing period at end, here and below.

Also, there are a bunch of methods above this one that appear to have missing args in their docstrings. Suggest fixing these as well.

Reply:
Period corrected. Missing docstrings for args pending.

Reply:
corrected missing period. Missing docstrings for args pending.